openrtb_ext/bidders.go

```diff
@@ -495,7 +509,7 @@ func CoreBidderNames() []BidderName {
 		BidderYieldone,
 		BidderZeroClickFraud,
 		BidderZetaGlobalSsp,
-	}
+	}, aliasBidderNames...)
```

Comment:
```go
}, AliasBidderNamesSlice()...)
```

Reply:
Not changing this because of comment above - https://github.com/prebid/prebid-server/pull/3027#discussion_r1295471270